Add sameness group field to prepared queries (#17089)
hashicorp/consul: website/content/api-docs/query.mdx

  - `Service` `(string: <required>)` - Specifies the name of the service to
    query.

  - `SamenessGroup` `(string: "")` <EnterpriseAlert inline /> - Specifies a Sameness group to forward the
  query to. The `SamenessGroup` will forward to its members in the order defined, returning on the first

>   healthy query. `SamenessGroup` is mutually exclusive with `Failover` as `SamenessGroup` members will be used
>   in place of a defined list of failovers.

Reviewer: ```suggestion
  healthy query. You can not use this field with `Failover` since Consul will use  the `SamenessGroup` members instead of the defined list of failovers.
```

  - `Namespace` `(string: "")` <EnterpriseAlert inline /> - Specifies the Consul namespace

>     to query. If not provided the query will use Consul default namespace for resolution. When combined with
>   `SamenessGroup` this will specify the namespaces in which the `SamenessGroup` will resolve all members listed.

Reviewer: ```suggestion
    to query. If not provided, the query will use Consul's default namespace for resolution. When combined with
  `SamenessGroup`, this will specify the namespaces in which the `SamenessGroup` will resolve all members listed.
```

  - `Partition` `(string: "")` <EnterpriseAlert inline /> - Specifies the Consul partition

>   to query. If not provided the query will use Consul default partition for resolution. When combined with
>   `SamenessGroup` this will specify the partition in which the `SamenessGroup` exists

Reviewer: ```suggestion
  to query. If not provided, the query will use Consul default partition for resolution. When combined with
  `SamenessGroup`, this will specify the partition in which the `SamenessGroup` exists
```

>   to query. If not provided the query will use Consul default partition for resolution. When combined with
>   `SamenessGroup` this will specify the partition in which the `SamenessGroup` exists

Reviewer: ```suggestion
  to query. If not provided, the query will use Consul default partition for resolution. When combined with
  `SamenessGroup`, this will specify the partition where the `SamenessGroup` exists.
```

>   - `Failover` Determines what happens if no healthy nodes are available in the local datacenter when

Reviewer: Can we provide the type for Failover?

Author: We don't include the type with the other "objects" which have their fields defined tabbed underneath

    the query is executed. It allows the use of nodes in other datacenters with

>     very little configuration. This field is mutually exclusive with `SamenessGroup`.

Reviewer: ```suggestion
    very little configuration. You can not use this field with `SamenessGroup`.
```
